ui/raidboss/raidboss_config.ts

```diff
@@ -273,16 +289,25 @@ const kMiscTranslations = {
   },
 };
 
-const validDurationOrUndefined = (val) => {
-  val = parseFloat(val);
-  if (!isNaN(val) && val >= 0)
-    return val;
+const validDurationOrUndefined = (val: string) => {
+  const parsed = parseFloat(val);
+  if (!isNaN(parsed) && parsed >= 0) return parsed;
   return undefined;
 };
 
-const canBeConfigured = (trig) => !trig.isMissingId && !trig.overriddenByFile;
+interface Trig extends BaseTrigger<Data> {
+  isMissingId: boolean;
+  overriddenByFile: boolean;
+}
```

Comment:
Looks liek there is only one line used this type, and technically this is not a Trigger property?
Instead of declare a new interface, what about to just `trig: BaseTrigger<Data> & { isMissingId?: boolean; overriddenByFile?: boolean }` ?